mount.cc

```diff
@@ -27,11 +27,19 @@ class MountWorker : public Nan::AsyncWorker {
   // This function is executed in another thread at some point after it has been
   // scheduled. IT MUST NOT USE ANY V8 FUNCTIONALITY.
   void Execute() {
-    int ret = mount(mounty->devFile.c_str(),
-            mounty->target.c_str(),
-            mounty->fsType.c_str(),
-            mounty->flags,
-            mounty->data.c_str());
+    #ifdef __linux__
+      int ret = mount(mounty->devFile.c_str(),
+              mounty->target.c_str(),
+              mounty->fsType.c_str(),
+              mounty->flags,
+              mounty->data.c_str());
+    #elif __APPLE__
+      int ret = mount(
+              mounty->fsType.c_str(),
+              mounty->target.c_str(),
+              mounty->flags,
+              &mounty->data);
```

Comment:
Does it accept a `string` object? I've seen it in other places... shouldn't them to be a pointer to `c_str`?